samples/core/src/main/resources/db/migration/mysql/V0001__R001_Create_schema_MYSQL.sql.mysql

```diff
@@ -0,0 +1,137 @@
+    pictureId BIGINT,
+    alcoholic BIT,
+    dType VARCHAR(31) NOT NULL,
+    id BIGINT NOT NULL,
+    rev BIGINT NOT NULL
+);
+
+-- *** Offer ***
+CREATE TABLE OFFER(
+    id BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    modificationCounter INT NOT NULL,
+    description VARCHAR(255),
+    name VARCHAR(255),
+    price DECIMAL(19, 2),
+    number BIGINT,
+    state INT,
+    drinkId BIGINT,
+    mealId BIGINT,
+    sideDishId BIGINT
+);
+
+ALTER TABLE OFFER ADD CONSTRAINT UC_OFFER_NAME UNIQUE(name);
+ALTER TABLE OFFER ADD CONSTRAINT UC_OFFER_NUMBER UNIQUE(number);
+
+ALTER TABLE OFFER ADD CONSTRAINT FK_OFFER2SIDEDISH FOREIGN KEY(sideDishId) REFERENCES PRODUCT(id);
+ALTER TABLE OFFER ADD CONSTRAINT FK_OFFER2MEAL FOREIGN KEY(mealId) REFERENCES PRODUCT(id);
+ALTER TABLE OFFER ADD CONSTRAINT FK_OFFER2DRINK FOREIGN KEY(drinkId) REFERENCES PRODUCT(id);
+
+-- *** RestaurantTable (Table is a reserved keyword in Oracle) ***
+CREATE TABLE RESTAURANTTABLE(
+    id BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    modificationCounter INT NOT NULL,
+    number BIGINT NOT NULL CHECK (NUMBER >= 0),
+    state INT,
+    waiterId BIGINT
+);
+
+ALTER TABLE RESTAURANTTABLE ADD CONSTRAINT UC_TABLE_NUMBER UNIQUE(number);
+
+-- *** RestaurantOrder (Order is a reserved keyword in Oracle) ***
+CREATE TABLE RESTAURANTORDER(
+    id BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    modificationCounter INTEGER NOT NULL,
+    state INT,
+    tableId BIGINT NOT NULL
+);
+
+-- *** OrderPosition ***
+CREATE TABLE ORDERPOSITION(
+    id BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    modificationCounter INT NOT NULL,
+    comment VARCHAR(255),
+    cookId BIGINT,
+    offerId BIGINT,
+    offerName VARCHAR(255),
+    price DECIMAL(19, 2),
+    state INT,
+    drinkState INT,
+    orderId BIGINT
+);
+
+ALTER TABLE ORDERPOSITION ADD CONSTRAINT FK_ORDPOS2ORDER FOREIGN KEY(orderId) REFERENCES RESTAURANTORDER(id);
+ALTER TABLE ORDERPOSITION ADD CONSTRAINT FK_ORDPOS2COOK FOREIGN KEY(cookId) REFERENCES STAFFMEMBER(id);
+
+-- *** Bill ***
+CREATE TABLE BILL(
+    id BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    modificationCounter INT NOT NULL,
+    payed BIT NOT NULL,
+    tip DECIMAL(19, 2),
+    total DECIMAL(19, 2)
+);
+
+CREATE TABLE BILLORDERPOSITION(
+    billId BIGINT NOT NULL,
+    orderPositionsId BIGINT NOT NULL
+);
+
+ALTER TABLE BILLORDERPOSITION ADD CONSTRAINT FK_BILLORDPOS2BILL FOREIGN KEY(billId) REFERENCES BILL(id);
+ALTER TABLE BILLORDERPOSITION ADD CONSTRAINT FK_BILLORDPOS2ORDPOS FOREIGN KEY(orderPositionsId) REFERENCES ORDERPOSITION(id);
+
+-- *** BinaryObject (BLOBs) ***
+CREATE TABLE BINARYOBJECT (
+  id BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  modificationCounter INT NOT NULL,
+  data LONGBLOB,
+  size BIGINT NOT NULL,
+  mimeType VARCHAR(255)
+);
+
+-- *** RevInfo (Commit log for envers audit trail) ***
+CREATE TABLE REVINFO(
+    id BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    timestamp BIGINT NOT NULL,
+    user VARCHAR(255)
```

Comment:
does this field "user" have to be aligned with the previous change to userLogin on the PR526 ?